src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationMiddleware.cs

```diff
@@ -40,6 +40,18 @@ public class CookieAuthenticationMiddleware : AuthenticationMiddleware<CookieAut
             {
                 Options.CookieManager = new ChunkingCookieManager(urlEncoder);
             }
+            if (!Options.LoginPath.HasValue)
+            {
+                Options.LoginPath = CookieAuthenticationDefaults.LoginPath;
+            }
+            if (!Options.LogoutPath.HasValue)
+            {
+                Options.LogoutPath = CookieAuthenticationDefaults.LogoutPath;
```

Comment:
I don't think this one is a redirect. This one just specified what uri will enable the return uri to trigger
A redirect so I think it's ok to have a default.

They still have to send a query string with return url on it to the actual logout path and call sign out there for this to have any effect